wowup-electron/src/app/components/common/client-selector/client-selector.component.ts

```diff
@@ -46,13 +46,16 @@ export class ClientSelectorComponent implements OnInit {
 
   private async mapInstallations(installations: WowInstallation[]): Promise<WowInstallation[]> {
     let total = 0;
-    for (const inst of installations) {
+
+    const displayedInstallations: WowInstallation[] = installations.filter((i) => !i.isHidden);
```

Comment:
We'll probably need a migration to set everything to `isHidden = false`, but I don't know how that should be added. @jliddev is that something you can explain or add later? Without the migration every client will suddenly be hidden.

Reply:
isHidden is optional and will be considered "false" if not present. So every client should be displayed by default and hidden only if manually set to in the options

Reply:
Ah my bad, it's the double negative